src/AdaptiveCurveIrm.sol

```diff
@@ -54,32 +42,10 @@ contract AdaptiveCurveIrm is IAdaptiveCurveIrm {
 
     /// @notice Constructor.
     /// @param morpho The address of Morpho.
-    /// @param curveSteepness The curve steepness (scaled by WAD).
-    /// @param adjustmentSpeed The adjustment speed (scaled by WAD).
-    /// @param targetUtilization The target utilization (scaled by WAD).
-    /// @param initialRateAtTarget The initial rate at target (scaled by WAD).
-    constructor(
-        address morpho,
-        int256 curveSteepness,
-        int256 adjustmentSpeed,
-        int256 targetUtilization,
-        int256 initialRateAtTarget
-    ) {
+    constructor(address morpho) {
         require(morpho != address(0), ErrorsLib.ZERO_ADDRESS);
-        require(curveSteepness >= WAD, ErrorsLib.INPUT_TOO_SMALL);
-        require(curveSteepness <= ConstantsLib.MAX_CURVE_STEEPNESS, ErrorsLib.INPUT_TOO_LARGE);
-        require(adjustmentSpeed >= 0, ErrorsLib.INPUT_TOO_SMALL);
-        require(adjustmentSpeed <= ConstantsLib.MAX_ADJUSTMENT_SPEED, ErrorsLib.INPUT_TOO_LARGE);
-        require(targetUtilization < WAD, ErrorsLib.INPUT_TOO_LARGE);
-        require(targetUtilization > 0, ErrorsLib.ZERO_INPUT);
-        require(initialRateAtTarget >= ConstantsLib.MIN_RATE_AT_TARGET, ErrorsLib.INPUT_TOO_SMALL);
-        require(initialRateAtTarget <= ConstantsLib.MAX_RATE_AT_TARGET, ErrorsLib.INPUT_TOO_LARGE);
```

Comment:
The information contained in those `require`s will be lost after this PR. Having tests that check the value of the constants exactly as those requires do + document those values would reduce future errors.